libs/damap/src/assets/styles.scss

```diff
@@ -154,3 +150,19 @@ mat-card {
 .button-position {
   margin: 5px;
 }
+
+.tooltip-position {
+  position: absolute;
+  right: 30px;
+  margin: 5px;
+}
+
+.button-position-custom {
+  color: var(--white-smoke-text) !important;
+  font-weight: 400 !important;
+}
```

Comment:
See also comment on the white-smoke-text variable.

Additionally, the button font could be set in general as well (I think this should be done in the core damap-frontend anyway). Then there would be no need to add this class to most buttons. If possible, a different solution would be preferable.

If we keep it, I would challenge the name of the class. It suggests it has something to do with the button position but that is not the case.

Reply:
Ah yes, this was planned only for those two buttons therefore i placed it in. But i used everywhere the --active color for button, but i forgot to clean that up, thanks for mentioning. The rest fits well green, just in those 'plus' buttons (home, plans and costs) needs to be white-smoke.